tests/test_integration/test_partner_resource.py

```diff
@@ -0,0 +1,35 @@
+from tests.test_integration.helpers import create_partner
+from tests.test_integration.test_case import YolaServiceTestCase
+
+
+class TestYolaPartners(YolaServiceTestCase):
+    """Yola: Partner resource"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(TestYolaPartners, cls).setUpClass()
+        cls.partner = create_partner(cls.service)
+        cls.partner_id = cls.partner['id']
+        # There's no way to really delete the test partners we're creating :(
+
+    def test_can_create_partner(self):
+        self.assertEqual(self.partner['properties']['website'], 'example.com')
```

Comment:
I find this an awkward test.

I would expect to see a service call here, forcing the test to document "how to create a partner".

Otherwise, seems like it's just verifying the helper (which we know works otherwise all the other tests would fail).